magento/magento2
# Fixes issue with backorders enabled and qty < 0, is_in_stock = 1

### Description (*)

This fixes an issue with importing a product that has qty <= 0 and is_in_stock = 1 and Back orders enabled 
### Fixed Issues (if relevant)

1. Fixes magento/magento2#32339

### Manual testing scenarios (*)

1. Run csv import with product that has qty <= 0 and is_in_stock = 1 and Back orders enabled 
2. Customer should be able to purchase the product as backorder


### Contribution checklist (*)
